src/App.tsx

```diff
@@ -11,11 +12,31 @@ import './components/common/styles/common-styles.scss';
 export class App extends React.PureComponent {
   private readonly wbState = new WordBashState();
 
+  componentDidMount() {
```

Comment:
Make WordBashState::onResize public, call it from here.
Then remove your versions from here.